From 7da1940f7fde35d053af38073362cb7b56996574 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Wed, 5 Jan 2022 15:41:29 +0100
Subject: [PATCH 0374/1695] plat-stm32mp2: conf: support FMC driver

Default enable FMC driver for platform stm32mp2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4d4c117350d4eacbec870c37476684264f50629b plat-stm32mp2: conf: support FMC driver]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: Iad5c5e90ec7d60b062a83218aea0066bc2d1c086
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/243078
Reviewed-by: Nicolas LE BAYON <nicolas.le.bayon@st.com>
Reviewed-by: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 34a2b11e3..194a99505 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -68,6 +68,7 @@ CFG_STM32MP_OPP_COUNT ?= 3
 
 CFG_STM32_BSEC3 ?= y
 CFG_STM32_EXTI ?= y
+CFG_STM32_FMC ?= y
 CFG_STM32_GPIO ?= y
 CFG_STM32_IAC ?= y
 CFG_STM32_RIF ?= y
-- 
2.47.2

